content/en/docs/tasks/kubelet-credential-provider/kubelet-credential-provider.md

```diff
@@ -0,0 +1,114 @@
+---
+title: Kubelet Image Credential Provider
+reviewers:
+- liggitt
+- cheftako
+description: Configure the kubelet's image credential provider plugin
+content_type: task
+---
+
+<!-- overview -->
+
+Starting from Kubernetes v1.20, the kubelet can dynamically retrieve credentials for a container image registry
+using exec plugins. The kubelet and the exec plugin communicate through stdio (stdin, stdout, and stderr) using
+Kubernetes versioned APIs. These plugins allow the kubelet to request credentials for a container registry dynamically
+as oppposed to storing static credentials on disk. For example, the plugin may talk to a local metadata server to retrieve
+short-lived credentials for an image that is being pulled by the kubelet.
+
+You may be interested in using this capabilitiy if any of the below are true:
+1. API calls to a cloud provider service are required to retreive authentication information for a registry.
+2. Credentials have short expiration times and requesting new credentials frequently is required.
+3. Storing registry credentials on disk or in image pull Secrets is not acceptable.
+
+This guide demonstrates how to configure the kubelet's image credential provider plugin mechanism.
+
+{{< note >}}
+This capabilitiy will eventually replace built-in logic in the kubelet to serve registry credentials for Azure, AWS and GCP.
+{{< /note >}}
+
+## {{% heading "prerequisites" %}}
+
+* kubelet on v1.20 or later. If on v1.20, ensure the alpha feature gate `KubeletCredentialProviders` is on.
```

Comment:
`KubeletCredentialProviders` needs to be added to the feature gate list `content/en/docs/reference/command-line-tools-reference/feature-gates/` with a short description describing the feature

Reply:
ack, thanks

Reply:
Great call @tengqm, will update with your suggestion